src/pycram/external_interfaces/giskard.py

```diff
@@ -2,26 +2,40 @@
 import threading
 
 import sys
+from typing import TYPE_CHECKING
 
+from ..object_descriptors.generic import NamedBoxVisualShape
 from ..ros import Time
 from ..ros import logwarn, loginfo_once, loginfo
 from ..ros import get_node_names
 
 from ..datastructures.enums import JointType, ObjectType, Arms
-from ..datastructures.pose import PoseStamped
+from ..datastructures.pose import PoseStamped, Vector3, Header, Vector3Stamped
 from ..datastructures.world import World
 from ..datastructures.dataclasses import MeshVisualShape
 from ..ros import get_service_proxy
 from ..world_concepts.world_object import Object
 from ..robot_description import RobotDescription
 
 from typing_extensions import List, Dict, Callable, Optional
+from geometry_msgs.msg import PoseStamped as ROSPoseStamped, PointStamped, QuaternionStamped, \
+    Vector3Stamped as ROSVector3Stamped
+
 from threading import Lock, RLock
 from pycram.ros import logging as log
 
+try:
+    from giskardpy_ros.python_interface.python_interface import GiskardWrapper as GiskardWrapper
+    from giskard_msgs.msg import WorldBody, MoveResult, CollisionEntry, LinkName
+    from giskardpy.motion_statechart.tasks.mixing import SpiralMixing
+    from giskardpy.motion_statechart.tasks.task import WEIGHT_BELOW_CA
+except ModuleNotFoundError as e:
+    logwarn("Failed to import Giskard messages, the real robot will not be available")
 
+if TYPE_CHECKING:
+    from giskardpy_ros.python_interface.python_interface import GiskardWrapper as GiskardWrapper
 
-giskard_wrapper = None
+giskard_wrapper: GiskardWrapper = None
```

Comment:
Doing that requires that Giskard is always installed (that's also why the CI is failing) I think it is better to move it in the try-catch block